cli/dev/register.ts

```diff
@@ -53,20 +53,30 @@ export const registerDev = (program: Command) => {
           console.log("Found entrypoint at:", entrypointPath)
         } else {
           console.log(
-            "No entrypoint found. Run 'tsci init' to bootstrap a basic project.",
+            kleur.yellow(
+              "No entrypoint found. Starting development server in sandbox mode.",
```

Comment:
```suggestion
              "No entrypoint found. Starting development server.",
```

Remove all the reference of `sandbox` from the other files too! It's not a sandbox